lib/p5.play.js

```diff
@@ -1371,28 +1339,19 @@ function Sprite(pInst, _x, _y, _w, _h) {
    * @method setDefaultCollider
    */
   this.setDefaultCollider = function() {
+    if(animations[currentAnimation] && animations[currentAnimation].getWidth() === 1 && animations[currentAnimation].getHeight() === 1) {
+      //animation is still loading
+      return;
+    }
 
     //if has animation get the animation bounding box
     //working only for preloaded images
-    if(animations[currentAnimation] && (animations[currentAnimation].getWidth() !== 1 && animations[currentAnimation].getHeight() !== 1))
-    {
-      this.collider = this.getBoundingBox();
+    if(animations[currentAnimation]) {
       this._internalWidth = animations[currentAnimation].getWidth()*abs(this._getScaleX());
       this._internalHeight = animations[currentAnimation].getHeight()*abs(this._getScaleY());
-      //quadTree.insert(this);
-      this.colliderType = 'image';
-      //print("IMAGE COLLIDER ADDED");
-    }
-    else if(animations[currentAnimation] && animations[currentAnimation].getWidth() === 1 && animations[currentAnimation].getHeight() === 1)
-    {
-    //animation is still loading
-    //print("wait");
-    }
-    else //get the with and height defined at the creation
-    {
-      this.collider = new AABB(pInst, this.position, createVector(this._internalWidth, this._internalHeight));
-      //quadTree.insert(this);
-      this.colliderType = 'default';
+      this.setCollider('rectangle');
+    } else {
+      this.setCollider('rectangle');
```

Comment:
Thanks to another change made earlier this month, `setCollider('rectangle')` is now pretty much the same thing as setting a 'default' collider - we default to an OBB matching the sprite dimensions, and expect it to update with the sprite.